server/sonar-server/src/main/java/org/sonar/server/issue/ws/IssuesWs.java

```diff
@@ -46,7 +46,7 @@ public IssuesWs(IssuesWsAction... actions) {
   @Override
   public void define(Context context) {
     NewController controller = context.createController(API_ENDPOINT);
-    controller.setDescription("Coding rule issues");
+    controller.setDescription("Read and update issues.");
```

Comment:
why not Manage Issues? we search for issues, which is an important element of the domain @ganncamp WDYT?